config.json

```diff
@@ -1,264 +1,1802 @@
-      "resource": "oil", 
-      "cities": 4
+      "resource_cost": 2, 
+      "resources": [
+        "oil"
+      ], 
+      "type": "light"
     }, 
     {
-      "resource_cost": 2, 
+      "cities": 3, 
       "cost": 18, 
-      "resource": "oil", 
-      "cities": 3
+      "resource_cost": 2, 
+      "resources": [
+        "oil"
+      ], 
+      "type": "light"
     }, 
     {
-      "resource_cost": 2, 
+      "cities": 2, 
       "cost": 10, 
-      "resource": "oil", 
-      "cities": 2
+      "resource_cost": 2, 
+      "resources": [
+        "oil"
+      ], 
+      "type": "dark"
     }, 
     {
-      "resource_cost": 1, 
+      "cities": 1, 
       "cost": 6, 
-      "resource": "oil", 
-      "cities": 1
+      "resource_cost": 1, 
+      "resources": [
+        "oil"
+      ], 
+      "type": "dark"
     }, 
     {
-      "resource_cost": 2, 
+      "cities": 7, 
       "cost": 50, 
-      "resource": "nuclear", 
-      "cities": 7
+      "resource_cost": 2, 
+      "resources": [
+        "nuclear"
+      ], 
+      "type": "light"
     }, 
     {
-      "resource_cost": 2, 
+      "cities": 6, 
       "cost": 37, 
-      "resource": "nuclear", 
-      "cities": 6
+      "resource_cost": 2, 
+      "resources": [
+        "nuclear"
+      ], 
+      "type": "light"
     }, 
     {
-      "resource_cost": 2, 
+      "cities": 5, 
       "cost": 32, 
-      "resource": "nuclear", 
-      "cities": 5
+      "resource_cost": 2, 
+      "resources": [
+        "nuclear"
+      ], 
+      "type": "light"
     }, 
     {
-      "resource_cost": 1, 
+      "cities": 3, 
       "cost": 21, 
-      "resource": "nuclear", 
-      "cities": 3
+      "resource_cost": 1, 
+      "resources": [
+        "nuclear"
+      ], 
+      "type": "light"
     }, 
     {
-      "resource_cost": 1, 
+      "cities": 2, 
       "cost": 13, 
-      "resource": "nuclear", 
-      "cities": 2
+      "resource_cost": 1, 
+      "resources": [
+        "nuclear"
+      ], 
+      "type": "dark"
     }, 
     {
-      "resource_cost": 2, 
+      "cities": 5, 
       "cost": 35, 
-      "resource": "oil&gas", 
-      "cities": 5
+      "resource_cost": 2, 
+      "resources": [
+        "oil", 
+        "gas"
+      ], 
+      "type": "light"
     }, 
     {
-      "resource_cost": 3, 
+      "cities": 5, 
       "cost": 22, 
-      "resource": "oil&gas", 
-      "cities": 5
+      "resource_cost": 3, 
+      "resources": [
+        "oil", 
+        "gas"
+      ], 
+      "type": "light"
     }, 
     {
-      "resource_cost": 2, 
+      "cities": 3, 
       "cost": 8, 
-      "resource": "oil&gas", 
-      "cities": 3
-    }
-  ],
-  "stage_three_card": 
-    {
-      "resource_cost": 0,
-      "cost": 99,
-      "resource": "stage three",
-      "cities": 0,
-      "type": "light"
+      "resource_cost": 2, 
+      "resources": [
+        "oil", 
+        "gas"
+      ], 
+      "type": "dark"
     }
+  ], 
+  "maps": {
+    "europe": [
```

Comment:
Added the map into the config too.  This way we can always add in USA later.  JSON converts tuples to lists (because JSON doesn't have a tuple as a structure it can use).  It's fine though because lists and tuples are _almost_ the same in python